packages/icons/package.json

```diff
@@ -25,12 +25,12 @@
     "chalk": "5.0.0",
     "node-emoji": "1.11.0",
     "npm-run-all": "4.1.5",
-    "react": "16.14.0",
+    "react": "17.0.2",
     "rimraf": "3.0.2",
     "typescript": "3.9.10"
   },
   "peerDependencies": {
-    "react": "^16.12.0"
+    "react": "^17.0.2"
   },
   "gitHead": "f9e7778c841194fa776c5dc45d62fb9caf408f22"
-}
+}
```

Comment:
nit: It looks like the trailing newline was removed.

Reply:
Yeah, that's weird. I thought I had that built into my editor. Thanks for the call out.